src/foam/nanos/notification/MyNotificationDAO.js

```diff
@@ -34,6 +34,7 @@ foam.CLASS({
         if ( user == null ) return sink;
         return getDelegate().where(MLang.AND(
             MLang.EQ(Notification.USER_ID, user.getId()),
+            MLang. EQ (Notification. IN_APP_ENABLED, true),
```

Comment:
This compiles?

Reply:
You betcha and it works. But I can fix it.